lesson22/main.js

```diff
@@ -120,7 +121,6 @@ const formingTableHeaderNameWithKey = (key) => {
  *Creating table data with value of usersData
  * @param {Array} usersData The Array of usersData
  */
-
```

Comment:
I was thinking "what is this..?" so I googled about it and then I just knew about JSDOC. I did not know about it so I lerned a lot!

Reply:
Thank you so much for reviewing! Ms.Onozuka told me about this before.
I'm happy to share about this with you :)